.github/workflows/bundle.yml

```diff
@@ -0,0 +1,60 @@
+name: Bundle
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  bundle:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+      - name: Install requirements
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r dev_requirements.txt
+      - name: Bundle (MacOS)
+        run: |
+          python -m nuitka --standalone --include-data-dir=source=source --noinclude-data-files='*.json' --remove-output --assume-yes-for-downloads --macos-create-app-bundle --macos-app-icon=source/resources/icon.png microcosm.py
+          ditto -c -k --sequesterRsrc --keepParent microcosm.app microcosm-osx.zip
+        if: matrix.os == 'macos-latest'
+      - name: Bundle (Ubuntu)
+        run: |
+          python -m nuitka --onefile --standalone --include-data-dir=source=source --noinclude-data-files='*.json' --remove-output --assume-yes-for-downloads --linux-icon=source/resources/icon.png -o microcosm microcosm.py
+        if: matrix.os == 'ubuntu-latest'
+      - name: Bundle (Windows)
+        run: |
+          python -m nuitka --onefile --standalone --include-data-dir=source=source --noinclude-data-files='*.json' --remove-output --assume-yes-for-downloads --windows-icon-from-ico=source/resources/icon.png microcosm.py
+        if: matrix.os == 'windows-latest'
+      - name: Upload artifact (MacOS)
+        uses: actions/upload-artifact@v3
+        with:
+          name: microcosm-osx
+          path: microcosm-osx.zip
+        if: matrix.os == 'macos-latest'
+      - name: Upload artifact (Ubuntu)
+        uses: actions/upload-artifact@v3
+        with:
+          name: microcosm-linux
+          path: microcosm
+        if: matrix.os == 'ubuntu-latest'
+      - name: Upload artifact (Windows)
+        uses: actions/upload-artifact@v3
+        with:
+          name: microcosm-windows
+          path: microcosm.exe
+        if: matrix.os == 'windows-latest'
```

Comment:
The command compiles the game just fine and runs well on Windows - saving included.

Only issue is that the version compiled via the GH pipeline seems to trip Windows Defender - my locally made one is fine though?

Weird

Reply:
OK so just got it built locally and tripped WD too. Weirdly it's virus-free when I don't include the arg `--windows-icon-from-ico`, otherwise it gets tripped with that param included (I didn't have imageIO installed the first time I did it and just yeeted that command).

Reply:
This post seems to be support it/offer some overlap as well: https://stackoverflow.com/a/65775139

Reply:
Okay so I used virustotal and scanned the original exe, plus a couple that I made tonight, and all three still show the same thing as the original despite the second one not having the icon and the third one not having imageio installed. Did you run the same exact command?

Reply:
Added the `--mingw64` flag, which seems to not result in a hit on defender.

Reply:
Apparently works all good now!